ledger-core-cosmos/src/cosmos/CosmosLikeAccount.cpp

```diff
@@ -80,24 +81,58 @@ namespace ledger {
                         return std::dynamic_pointer_cast<CosmosLikeAccount>(shared_from_this());
                 }
 
-                // FIXME Review this!
+                static void computeAmount(CosmosLikeOperation &out, const cosmos::Message &msg) {
```

Comment:
The function is also a setter, so `computeAndSetAmount` would be clearer as a name in my opinion